src/amber/router/session/cookie_store.cr

```diff
@@ -0,0 +1,80 @@
+module Amber::Router::Session
+  # This is the default Cookie Store
+  class CookieStore < AbstractStore
+    property secret : String
+    property key : String
+    property expires : Int32
+    property store : Amber::Router::Cookies::Store
+    property session : Hash(String, String)
+
+    def initialize(@store, @key, @expires, @secret)
+      @session = current_session
+      @session[key] = SecureRandom.uuid
+    end
+
+    def id
+      @session[key]
+    end
+
+    def destroy
+      @session.clear
+    end
+
+    def [](key)
```

Comment:
The current session hash allows symbols or strings which saves typing and will be familiar to people coming from rails. Lets discuss the current SessionHash further before removing that.